[2024-11-05T13:26:46.722+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-05T13:26:46.767+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_catchup_backfill_v1.task1 scheduled__2024-11-02T00:00:00+00:00 [queued]>
[2024-11-05T13:26:46.798+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_catchup_backfill_v1.task1 scheduled__2024-11-02T00:00:00+00:00 [queued]>
[2024-11-05T13:26:46.799+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-11-05T13:26:46.888+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): task1> on 2024-11-02 00:00:00+00:00
[2024-11-05T13:26:46.899+0000] {standard_task_runner.py:72} INFO - Started process 220 to run task
[2024-11-05T13:26:46.904+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'dag_with_catchup_backfill_v1', 'task1', 'scheduled__2024-11-02T00:00:00+00:00', '--job-id', '63', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_catchup_backfill.py', '--cfg-path', '/tmp/tmpuskz5eum']
[2024-11-05T13:26:46.908+0000] {standard_task_runner.py:105} INFO - Job 63: Subtask task1
[2024-11-05T13:26:47.145+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_catchup_backfill_v1.task1 scheduled__2024-11-02T00:00:00+00:00 [running]> on host d1868bef5e0d
[2024-11-05T13:26:47.394+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Adithya' AIRFLOW_CTX_DAG_ID='dag_with_catchup_backfill_v1' AIRFLOW_CTX_TASK_ID='task1' AIRFLOW_CTX_EXECUTION_DATE='2024-11-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-02T00:00:00+00:00'
[2024-11-05T13:26:47.397+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-05T13:26:47.439+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-11-05T13:26:47.442+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo Hello my name is Adithya']
[2024-11-05T13:26:47.463+0000] {subprocess.py:86} INFO - Output:
[2024-11-05T13:26:47.466+0000] {subprocess.py:93} INFO - Hello my name is Adithya
[2024-11-05T13:26:47.467+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-11-05T13:26:47.562+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-05T13:26:47.563+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_with_catchup_backfill_v1, task_id=task1, run_id=scheduled__2024-11-02T00:00:00+00:00, execution_date=20241102T000000, start_date=20241105T132646, end_date=20241105T132647
[2024-11-05T13:26:47.647+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-05T13:26:47.689+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-05T13:26:47.694+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
